test/test_loc_handler.erl

```diff
@@ -0,0 +1,31 @@
+%% Copyright (C) 2013-2023, Travelping GmbH <info@travelping.com>
```

Comment:
this is new code, it should therefore be (C) 2023 only

Reply:
ok